projects/views.py

```diff
@@ -1,6 +1,22 @@
 from .models import Project
 
+from django.shortcuts import render
 from django.views.generic.list import ListView
+from django.http import HttpResponseRedirect
+
+from .forms import UploadForm
+
+
+def get_project(request):
```

Comment:
I think you can use a class-based view as well https://docs.djangoproject.com/en/1.8/topics/class-based-views/generic-editing/